Library/Formula/surfraw.rb

```diff
@@ -12,6 +12,9 @@ class Surfraw < Formula
     sha1 "b17ead2b3f8030e659d0af4b44332f76397607f5" => :mountain_lion
   end
 
+  depends_on "autoconf" => :build
```

Comment:
If these are just needed for `head` installations you should restrict the dependencies to `head`.

```ruby
head do
  url "git://git.debian.org/surfraw/surfraw.git", :shallow => false

  depends_on "autoconf" => :build
  depends_on "automake" => :build
end
```